Buildings/Controls/OBC/ASHRAE/G36/AHUs/MultiZone/SetPoints/ReliefDamper.mo

```diff
@@ -0,0 +1,143 @@
+﻿within Buildings.Controls.OBC.ASHRAE.G36.AHUs.MultiZone.SetPoints;
+block ReliefDamper "Control of actuated relief  dampers without fans"
+  parameter Modelica.SIunits.Pressure buiPreSet=0.05*248.84
+    "Building static pressure setpoint"
+    annotation(Evaluate=true);
+  parameter Real kp(min=0, unit="1") = 0.5
+    "Gain factor"
+    annotation(Dialog(group="Relief damper P-control parameter"));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput uBuiPre(
+    final unit="Pa",
+    quantity="PressureDifference")
+    "Measured building static pressure"
```

Comment:
Please write "Measured building static pressure difference" to make sure it is the difference rather than the absolute static pressure